src/main/java/com/github/hcsp/io/FileAccessor.java

```diff
@@ -1,23 +1,89 @@
 package com.github.hcsp.io;
 
+import org.apache.commons.io.FileUtils;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
 import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 
 public class FileAccessor {
-    public static List<String> readFile1(File file) {}
+    public static List<String> readFile1(File file) throws IOException {
+        InputStream inputStream=new FileInputStream(file);
+        List<String> list =new ArrayList<>();
+        String line="";
+        while (true){
+           int b= inputStream.read();
+           if (b==-1){
+               break;
+           }
+            if (b != '\n') {
+                line = String.join("", line, Character.toString((char) b));
```

Comment:
这样，每次都会新建一个字符串对象，它是不可变的，因此非常非常的没有效率。请使用`StringBuilder`来替代，`StringBuilder`是一个变长的对象，可以灵活的添加字符。
```suggestion
                line.append(b);
```